README.md

```diff
@@ -66,7 +67,7 @@ A valid (read: that follows this interface) IPFS core implementation, must expos
 
 `callback` must follow `function (err, node) {}` signature, where `err` is an error if the operation was not successful and `node` is a MerkleDAG node of the type [DAGNode][]
 
-If no `callback` is passed, a promise is returned.
+Returns a [promise][].
```

Comment:
I would rather for it to be one or the other. Why can't we reach out to the dev of promisify-es6 and understand the reason behind his decision? Seems like the dev might have expected that if someone passes a callback, to not grab the promise as well, there would be nothing good coming from that.